### DISCUSSION TITLE
small bug

### DISCUSSION THREAD

**Discussion by snr1999** [aeharding/ppg.report#148] [Q&A] [answered]
hello,
The project is great. Just what I was looking for. I love it!
I have small issue, that may be easy to repair, but is a little frustrating :)
The website does not save my speed setting. I want it to be "m/s" but every time I enter, it resets to km/h. It saves all other units, but not this one. I guess it because of "/" in saved value, but maybe not. Anyway maybe you would like to update this little bug for me.
best. kuba

**Comment by aeharding** [comment, marked as answer]
Thanks for the report! I created #149
